Rakefile

```diff
@@ -735,6 +738,31 @@ def github_access_token_query
   end
 end
 
+# @return [Boolean] Whether the empty Master section was added to the CHANGELOG
+#
+def add_empty_master_changelog_section(repo)
+  Dir.chdir(repo) do
+    changelog_file = 'CHANGELOG.md'
+    changelog = File.read(changelog_file)
+    return if changelog.match(/^## (.+)/).captures.first == 'Master'
+    return unless current_branch == 'master'
+    # This uses an array of strings so text editors dont strip the trailing spaces
+    changelog.sub! '##', ['## Master',
+                          '',
+                          '##### Enhancements',
+                          '',
+                          '* None.  ',
```

Comment:
What's the practise here, the first PR to add an enhancement will replace the None?